python/ray/autoscaler/_private/autoscaler.py

```diff
@@ -452,6 +470,38 @@ def set_prometheus_updater_data(self):
                 num_recovering += 1
         self.prom_metrics.recovering_nodes.set(num_recovering)
 
+    def _report_pending_infeasible(self, resources: List[ResourceDict]):
+        """Emit event messages for infeasible or unschedulable tasks."""
+        pending = []
+        infeasible = []
+        for bundle in resources:
+            placement_group = any("_group_" in k for k in bundle)
+            if placement_group:
+                continue
+            if self.resource_demand_scheduler.is_feasible(bundle):
+                pending.append(bundle)
+            else:
+                infeasible.append(bundle)
+        if pending:
+            if self.load_metrics.cluster_full_reported:
+                for request in pending:
+                    self.event_summarizer.add_once_per_interval(
+                        "Warning: The following resource request cannot be "
+                        "scheduled right now: {}. This is likely due to all "
+                        "cluster resources being claimed by actors. Consider "
+                        "creating fewer actors or adding more nodes "
+                        "to this Ray cluster.".format(request),
```

Comment:
why this only says "actors" not tasks?

Reply:
Also, why not mention that these are pending and that they will be scheduled later?

Reply:
We only print this if actors are there (tasks are considered transient and will finish eventually, so should not generate a warning).